pulsar-functions/api-java/pom.xml

```diff
@@ -42,6 +42,12 @@
       <artifactId>typetools</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.pulsar</groupId>
```

Comment:
@sijie @wolfstudy the concern comes back to whether the functions api should depend of the pulsar-client-api

Reply:
@jerrypeng let's stay in #4127 for the discussion